app/assets/stylesheets/admin/pages/_evaluations.scss

```
@@ -29,14 +32,9 @@
 
   .evaluation--suivi-par {
     margin-left: 1.75rem;
-    margin-top: .5rem;
-  }
-
-  .evaluation--parcours-type {
-    margin-top: 0.625rem;
   }
 
-  .evaluation--suivi-par, .evaluation--parcours-type {
+  .evaluation--suivi-par, .evaluation--parcours-type, .evaluation--created_at {
```

Ce ne sont pas des variations du composant evaluation mais des éléments du componsant, on utilise donc `__` de la méthodo BEM.
```suggestion
  .evaluation__suivi-par, .evaluation__parcours-type, .evaluation__created_at {
```